### PR TITLE
Fix issue #329: Windows indefinite loop

### LINKED ISSUE #329: Windows: Tanka not running on windows
When running "tk show environments/loki"

The process would keep running and not end

### PR DESCRIPTION
**Pull request by c0ffeec0der**
Problem: Indefinite loop in function findRoot

Root cause: Root path only consider linux "/"

Solution: Detect os at runtime and decide root path. Windows "C:\\", others "/"

Fixes #329

### CODE REVIEW

**Comment by malcolmholmes** [comment]
This makes the assumption that tanka files are on drive C:, is there another way to do it?

**Comment by c0ffeec0der** [comment]
Ah yes, good one! Code updated with _filepath.VolumeName_

File: pkg/jsonnet/jpath/jpath.go [hunk 1]
```diff
@@ -63,14 +64,22 @@ func Resolve(workdir string) (path []string, base, root string, err error) {
 // - tkrc.yaml is considered first, for a jb-independent way of marking the root
 // - if it is not present (default), jsonnetfile.json is used.
 func findRoot(start string) (dir string, err error) {
+  // root path based on os
+	var stop string
+	if runtime.GOOS == "windows" {
+	  stop = filepath.VolumeName(start) + "\\"
+	} else {
+		stop = "/"
+	}
```
Comment (sh0rez) [inline pkg/jsonnet/jpath/jpath.go:73]:
```suggestion
	stop := "/"
	if runtime.GOOS == "windows" {
	  stop = filepath.VolumeName(start) + "\\"
	}
```

**Comment by c0ffeec0der** [comment]
Hello,

Is there still anything required for this PR to be merged?

Thank you

**Pull request merged**